test/compliance_tool/test_compliance_check_aasx.py

```diff
@@ -6,12 +6,36 @@
 # SPDX-License-Identifier: MIT
 import os
 import unittest
+import zipfile
+
+import pyecma376_2
 
 from basyx.aas.compliance_tool import compliance_check_aasx as compliance_tool
 from basyx.aas.compliance_tool.state_manager import ComplianceToolStateManager, Status
 
 
 class ComplianceToolAASXTest(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
```

Comment:
Since a new instance of `ComplianceToolAASXTest` is created for every test, these files will zipped multiple times (once for each test method), resulting in unncessary overhead: https://docs.python.org/3/library/unittest.html#organizing-test-code
Instead, `setUpClass()` may be used to create the zip archives only once: https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass

Reply:
Implementing it right now actually. Thanks for the advise though:)

Reply:
As other tests also need AASX files, we cannot generate  AASX files in ComplianceToolAASXTest, because other tests can be run before or without tests from ComplianceToolAASXTest. So I moved the generation to __init__.py